opendbc/car/gm/values.py

```diff
@@ -58,6 +58,37 @@ def __init__(self, CP):
     self.BRAKE_LOOKUP_BP = [self.ACCEL_MIN, max_regen_acceleration]
     self.BRAKE_LOOKUP_V = [self.MAX_BRAKE, 0.]
 
+class WMI:
+    class CHEVROLET:
+        TRUCK = {"1GC", "2GC", "3GC"}
+        MPV = {"1G1", "1GN", "2G1", "2GN", "3G1", "3GN", "KL1", "KL7"}
+
+    class GMC:
+        TRUCK = {"1GT", "2GT", "3GT"}
+        MPV = {"1GK", "2GK", "3GK"}
+
+    class CADILLAC:
+        MPV = {"1G6", "1GY", "2G6", "3GY"}
+
+class ANY:
+  ALL = {
+    "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M",
+    "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z",
+    "0", "1", "2", "3", "4", "5", "6", "7", "8", "9"
+    }
+
+class YEARS:
+    _2016 = "G"
+    _2017 = "H"
+    _2018 = "J"
+    _2019 = "K"
+    _2020 = "L"
+    _2021 = "M"
+    _2022 = "N"
+    _2023 = "P"
+    _2024 = "R"
+    _2025 = "S"
```

Comment:
```suggestion
    S_2025 = "S"
```

this if anything. note Rivian has the same class, so we should make it common